src/App.tsx

```diff
@@ -20,12 +25,15 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                onChange={
+                  ({ target }) => setSearch(() => target.value.toLowerCase())
+                }
```

Comment:
For me destructuring and manipulations width data in layout complicates reading
(event) => setSearch(event.target.value) is easier 
and you can format your input in filter. Try it, may be you'll like it too )

Reply:
too much for a single callback, you'd rather create a separate function-handler
agree, such a structure like this is too much
```
onChange={
  callbackA => function(callbackB)
}

```